### ISSUE TITLE
Leather jackets and other gear, encumbrance incongruities

### ISSUE THREAD

**Issue by anoobindisguise** [CleverRaven/Cataclysm-DDA#66048]
### Is your feature request related to a problem? Please describe.

Leather Jacket: 1.5mm thickness, 5-7 torso encumbrance, 5 arms encumbrance, 97% torso coverage, 100% arms coverage, 1.6 liters storage across 2 pockets. [1.25 encumbrance per liter storage]

Red Leather Jacket: 1mm thickness, 7-10 torso encumbrance, 7 arms encumbrance, 97% torso coverage, 100% arms coverage, 1.6 liters storage across 2 pockets [1.5 encumbrance per liter storage]

Leather Police Jacket: 1.5mm thickness, 12-17 torso encumbrance, 5 arms encumbrance, 97% torso coverage, 65% arms coverage, 1.4 liters storage across 2 pockets. [3.6 encumbrance per liter storage]

Armored Leather Jacket: 1.5mm leather + 1.5mm budgetsteel, 10-18 torso encumbrance, 12 arms encumbrance, 97%+  73% steel torso coverage, 100% + 49% steel arms coverage, 1.2 liters storage across 4 pockets [6.7 encumbrance per liter storage]

Tire Leather Jacket: 1.5mm leather + ~2mm rubber, 12-20 torso encumbrance, 16 arms encumbrance, 97%+  68% rubber torso coverage, 100% + 51% rubber arms coverage, 1.2 liters storage across 4 pockets [6.7 encumbrance per liter storage]

The jackets have wildly different values, with Red Leather having more encumbrance for less protection than the vanilla jacket and Police having much more encumbrance and less coverage and less storage for the same protection. The armored jackets have extreme amounts of encumbrance especially when loaded with items in their pockets, which inexplicably become differently sized and more numerous (and much less storage overall). In addition all jackets have a weird "97%" torso coverage which might account for some gaps in the collar but a lot of jackets aren't like this and are fitted entirely up to the neck (casual jackets which aren't like this also are thinner at ~0.75mm coverage). 

Armored leather vest has the same issue, gaining not only vastly more encumbrance overall but also growing extra pockets, pockets becoming smaller, and costing WAY more encumbrance:liter for no reason.

Leather vest: 1.5mm thickness, 2-4 torso encumbrance, 90% torso coverage, 2.05 liters storage across 3 pockets. [0.98 encumbrance per liter storage]

Armored Leather vest: 1.5mm thickness + 1.5mm steel, 7-15 torso encumbrance, 90% torso coverage + 67% steel, 1.9 liters storage across 4 pockets. [4.2 encumbrance per liter storage] 

Meanwhile the leather trenchcoat and especially leather duster have more coverage and more storage and more storage:encumbrance than all of these.

### Solution you would like.

Gear should be congruous and also not become 5x worse at storing items by having steel plates sewn on.

### Describe alternatives you have considered.

The issue is that a lot of these have funky values (IRL jackets can be close enough to 1.5mm, but not all are like this) and I don't know how best to resolve some of these issues so I figured I'd just post an issue about it and see what people say.

**Comment by martosss** [comment]
- Regarding weird coverage, 97% seems perfectly reasonable once you understand what percentage of coverage it has on each subpart.
Those jackets cover 95% of the upper torso(which is 50% of the whole), and 100% of the lower torso(which is also 50% of the whole torso). If you average that you get 97.5% of the whole torso, so 97 when you round it down. Maybe the design idea is that it has a zipper/collar and that decreases coverage a bit. I wouldn't argue with it since I've had jackets irl that don't cover my neck and I need a scarf or something together with them for the winter to keep warm.

- Regarding encumbrance, I partially agree with you, but still, you shouldn't mix apples with oranges.
- you have encumbrance due to storage
- you have encumbrance due to protection

Now comparing the armored leather vest with the leather jacket...the 1st gives roughly 3x more protection than the leather jacket, so it's perfectly fair to me that it "costs" a little more encumbrance, because its main function would be to protect you and the pockets are bulky on top of that protection.

- Having said that, comparing the encmbrance of red leather vs the normal leather jacket has me confused... my best attempt to explain it would be that all else equal, the red leather has thinner material, which means it should be bulkier to compensate for that? Game logic... I personally cut up all of them for leather patches and use aprons/other vests, so I agree with you that their encumbrance is unrealistic.
And mixing storage with encumbrance is usually bad for gameplay - you don't need both - you're either fighting or looting, so either wear a heavy armor and no storage and go around killing stuff, or, once done with the killing, wear light clothes and a huge backpack/cart behind you and loot stuff. Mixing those 2 activities seems like an unnecessary burden that hampers both activities.

You could argue it depends on game style ... sometimes you can't kill a whole area, so you have to sneak in carefully and loot and run... and then if you meet an enemy you have to be prepared to fight ... but again, if you think you're gonna fight, just go and do it properly - kill everything first, then loot in peace - that way you're stronger while fighting, because you don't have that unnecessary encumbrance from pockets that are full with useless junk... and then while you loot you're not boiling in your armor, nor does it weigh on you, nor does it hamper your movement/foraging speed(legs,arms,feet encumbrance) - just take it off and go naked!

At this point you shouldn't question the rules, just play around them and have fun.
If a specific piece of armor looks useless, just don't wear it - use it for spare repair materials or trade it or burn it(well, don't burn it, really!) ... or use it as ingredient to craft other more useful things.

After playing for a while you should know that most clothes are only good for spare materials - that's what consumerism in the past lead to! :laughing:

**Comment by fairyarmadillo** [comment]
I would love to see armored leather jackets become more usable, as they're very on-theme. There's currently an issue where the medieval gear is all quite good but anything made of more modern materials really isn't, and it feels kind of awkward because our postapocalyptic wasteland is filled with players doing ren faire reenactment. In many cases there shouldn't really be a significant difference in performance, or if there is it should be slight. The armored jacket for instance requires 3 tailoring and is using a durable and comfortable factory-made item as its base, while the leather armor cuirass is hand-stitched, often out of scrap leather, and is only a 4 tailoring item.

I can accept that a cuirass is designed to be armor from the ground up for the purpose may have less bulk and weight than something jury-rigged, but the difference as it stands is pretty extreme.

One solution might be to identify items which are basically functioning in the same manner as existing pieces of medieval armor and adjust their stats to be in. An armored leather jacket is not dissimilar to a medieval coat of plates. The difference could be that one was made from a biker jacket and the other was made from scratch. In that case, the armored leather jacket could have slightly worse coverage or encumbrance, but the tradeoff would be that it was much easier to make, since most of the work is already done for you.

I know we don't have a coat of plates, but you get the idea.

**Comment by randomtyper** [comment]
> I would love to see armored leather jackets become more usable, as they're very on-theme. There's currently an issue where the medieval gear is all quite good but anything made of more modern materials really isn't, and it feels kind of awkward because our postapocalyptic wasteland is filled with players doing ren faire reenactment instead of Mad Max. In many cases there shouldn't really be a significant difference in performance, or if there is it should be slight. The armored jacket for instance requires 3 tailoring and is using a durable and comfortable factory-made item as its base, while the leather armor cuirass is hand-stitched, often out of scrap leather, and is only a 4 tailoring item.
> 
> I can accept that a cuirass is designed to be armor from the ground up for the purpose may have less bulk and weight than something jury-rigged, but the difference as it stands is pretty extreme.
> 
> One solution might be to identify items which are basically functioning in the same manner as existing pieces of medieval armor and adjust their stats to be in. An armored leather jacket is not dissimilar to a medieval coat of plates. The difference could be that one was made from a biker jacket and the other was made from scratch. In that case, the armored leather jacket could have slightly worse coverage or encumbrance, but the tradeoff would be that it was much easier to make, since most of the work is already done for you.
> 
> I know we don't have a coat of plates, but you get the idea.
> 
> > And mixing storage with encumbrance is usually bad for gameplay - you don't need both - you're either fighting or looting, so either wear a heavy armor and no storage and go around killing stuff, or, once done with the killing, wear light clothes and a huge backpack/cart behind you and loot stuff. Mixing those 2 activities seems like an unnecessary burden that hampers both activities.
> 
> I have to strongly disagree here. There are many armor items that I frequently use because they afford some storage, even if they're worse as armor than similar pieces. A good example is the survivor trenchcoat - it allows me to store essentials like ammo and medical supplies, and I can do some light looting without totally wrecking my ability to fight in melee. Storage/encumbrance does not need to be an all or nothing proposition and many players don't like fussing with shopping carts or backpacks all the time. I for one find it really nice to not have to make two trips, and I enjoy being more selective with my loot.
> 
> Given that not all armor pieces even have storage, I think the current model leaves the choice up to the player in a satisfying manner. If you do want to deal with backpacks and shopping carts, then you're not penalized - you can simply wear the pocketless armor.

1. People pre-Cataclysm didn't have to deal with zombies, fungi, triffids and eldritch horrors on daily basis while having to loot everything not nailed to walls. The civilian gear is mostly that - civilian. It is mostly good against weather. For protective and overall survival gear... Well, you have sporting gear like wetsuits and travelling backpacks, firefighter gear (probably the most useful for apocalypse civilian stuff for a reason), fencing gear and hunting gear.
2. Which leads to next point: you are basically fastening a bunch of scrap to a clothing item. Yes, the leather jacket may be comfortable and unencumbering when worn, but not when it is loaded with a bunch of misshapen metal. Hell, there is no metal related tool requirements besides hammering! IRL if you wanted to make a decent protective jacket that way you would need at least a hole puncher, just to make valid points to put the fasteners through.
4. Contrary to that, medieval armor (especially plate pieces) is probably the best protection for fighting zombies you can reasonably make. After all, it was made to stop swords, spears and arrows, let alone zombie bites and scratches. And given that you rarely meet heavy gun-armed opponents and the huge monsters make quite a lot of armor moot... It's either knight cosplay or full dodge build... And weak guns are still dangerous, so here I go stomping peasants. Again.
5. Leather armor you took as example is made by hand for YOU by YOU, knowing YOUR exact proportions and made to fit YOU. It is hand-stitched, yes, but it is hand-stitched for a specific person. There is a reason people even now go to tailors to order custom made-to-order clothes, after all.
6. I understand that scrap jackets a la Mad Max are cool and romantic, but CDDA tries to stay REALLY close to ground and reality currently. Maybe even too close. Regardless, in case of survival scenario IRL a well-fitted thick leather (almost rigid) piece that covers my whole torso would be vastly preferrable to having a bunch of scrap dangling from my jacket. Chains are cool, but they give you no tactical advantage whatsoever when they are just dangling from your belt. My fitted cuirass, on the other hand... *does a cartwheel into a roll*
7. Coat of plates is a coat of plates. It is made from the ground up to be an armor item, ingame brigandines (which are essentially coats of plate with smaller plates) are basically made from scratch including hammering out every plate and fitting them into a fully covering yet flexible armor piece (the crafting time is pretty long for one made of 1mm plates, more than 3 times of armored jackets, and tool and proficiency requirements are a "bit" more obscure). While armored jacket is basically a bunch of scrap hastily hammered and strapped onto a jacket.
8. Finally, for your concern of "There's currently an issue where the medieval gear is all quite good but anything made of more modern materials really isn't...". The plate armors are not really "medieval". Yes, the design is medieval (and tried and tested), but high quality steel became easily producible only in modern times. And there is plenty of good stuff made from modern materials. Like almost everything with Nomex in it. Or Kevlar (just find and kill a soldier zombie, wash his vest, take out the plates, and you are set for quite a while, take his helmet too, if you want). Neoprene is also neat, being good against bash and not too shabby against other sources of damage, while having decent warmth, good environmental and being water friendly as a bonus. Finally, here is an elephant in the room. Survivor gear. Essentially what you want to wear under your plate or instead of it in case of dusters and trenchcoats. Not salvaged leatherwear with metal scrap slapped on. And you mentioned it yourself that survivor trenchcoat has a useful combination of storage and low encumbrance (and I'd say its protection is not to be underestimated, too). Ah, yes, there is nylon medieval underarmor, too. It's not too shabby, i'd say. Calling everything made of modern materials not good is just a biased argument from your side.

**Comment by randomtyper** [comment]
That said, I agree that leather jackets (at least unmodified ones) need to be made to be uniform. Maybe give some elbow reinforcements to normal and red jacket, give police jacket a bit bigger pockets and stylish tag (the world has Mad Max fiction in it, after all it is our (if a bit different) Earth, so I think it would be pretty neat and logical). I wouldn't give the armored leather jacket anything more than pocket uniformisation and reduction of encumbrance per liter, though. It already gets a lot for little work (mild steel brigandine level of protection with drastically lower crafting requirements and time (which compensates for the coverage somewhat), warmth and pockets).

**Comment by randomtyper** [comment]
> I think you may have missed the thrust of my argument. An armored leather jacket is something like this, but with sleeves (and less costumey, but you get the idea): ![image](https://user-images.githubusercontent.com/94415528/243937538-cbfe6237-8bb9-4f4f-9756-7403cfd15bf7.png)
> 
> Yet it has almost double the base torso encumbrance of mild steel bringandine, which looks like this: ![image](https://user-images.githubusercontent.com/94415528/243937793-becc8885-bdae-43e6-a0e0-749e1987375a.png)
> 
> The latter item should be better in all respects including encumbrance as it's purpose-built and not jury-rigged, but it should not be twice as easy to move in. As it is, the encumbrance on the armored leather jacket is prohibitively high.
> 
> The armored leather jacket's torso encumbrance is also double what the armored leather vest's is, despite the fact that they're made of the same material at the same thickness and nearly the same coverage. That definitely feels like an oversight to me, and is generally indicative of the inconsistent state of the non-medieval items.

I'd say the leather jacket you linked looks a bit too well-made for something that only requires hammering and random scrap. Something like this https://www.pinterest.com/pin/847239748636727211/ or this https://rust.fandom.com/wiki/Road_Sign_Jacket would be much more representative for something with that time and crafting tool and material requirements (I am pretty sure half of those 7 hours are spent sorting and hammering out passable scrap). Wanna make the encumbrance lower? Add chisel as tool requirement and more string. Plus more hours to make (about 10 with all proficiencies). Or make material thinner.

**Comment by fairyarmadillo** [comment]
The roadsign vest from Rust is not at all representative of the CDDA item, which is specifically a biker jacket with scrap metal attached to it.

The armored leather jacket requires tailoring 4, garment closures, and skilled leatherworking. That's quite a bit!

You're using inconsistent logic. If sleeves add torso encumbrance, then why isn't that the case with all armor, or even really any of it except this one piece?

**Comment by randomtyper** [comment]
> The roadsign vest from Rust is not at all representative of the CDDA item, which is specifically a biker jacket with scrap metal attached to it.

The CDDA item is specifically a biker jacket with scrap metal TIED to it without any holes made in metal. That is even worse than roadsign vest, I'd say.
 
> The armored leather jacket requires tailoring 4, garment closures, and skilled leatherworking. That's quite a bit!

Because survivor screws with pockets alongside with strapping scrap for some reason. Remove those requirements then.

> You're using inconsistent logic. If sleeves add torso encumbrance, then why isn't that the case with all armor, or even really any of it except this one piece?

Alright, got me on that one. Still, I am in favor of putting the encumbrance of both on a value higher than encumbrance of the vest. And normalising coverage of vest to 97 of torso like on jacket (along with standardising pockets), so it is reasonable to be a jacket with torn off sleeves. 

> Scrap cuirass/suits already exist to represent a bunch of random metal awkwardly tied together. The armored jacket can easily occupy a middle space between complete junk and something more sensible, which would give it a use case it presently lacks. If that means amping up the requirements, so be it - nobody but the NPC bandits are using the things currently anyway.

The scrap cuirass does not give any storage nor significant warmth. And if you actually look into material stats, leather jacket is more protective than scrap cuirass in all areas except blunt. And it's not like scrap suit is awkwardly tied together, really. It has a full cloth backing and it is implied that there are essentially pockets for the metal sheets (like, you place a metal sheet on cloth sheet and sew on cloth patches to cloth sheet while overlapping every corner of the metal sheet so the metal does not fall off).

**Comment by estebandellasilva** [comment]
If someone takes a look at encumbrance values also please check the zentai + lycra body suit ... somethings weird there kinda
Zentai covers Head, Mouth, Eyes, Torso, Arms, Hands, Legs, Feet
warmth 20 thickness 0.1mm Volume 0,5l Weight 0,6kg encumbrance 0
Lycra Body suit covers  Torso, Arms, Hands, Legs, Feet
warmth 15 thickness 1,0mm Volume 2,0l Weight 0,41kg encumbrance 14 
(stats taken from hitchhiker ingame encumbrance is 7 if its not poor fit)

How can the bodysuit which has 10 times the thickness of material less weight than the zentai less warmth than the zentai 
Also 7 encumbrance for a skintight suit seem a lot considering that lycra should not hinder movement that much
basically zentai is kinda too good but lycra body suit iss kinda bad for no reason - protection seems a little high though

**Comment by randomtyper** [comment]
> If someone takes a look at encumbrance values also please check the zentai + lycra body suit ... somethings weird there kinda
> 
> Zentai covers Head, Mouth, Eyes, Torso, Arms, Hands, Legs, Feet warmth 20 thickness 0.1mm Volume 0,5l Weight 0,6kg encumbrance 0
> 
> Lycra Body suit covers Torso, Arms, Hands, Legs, Feet warmth 15 thickness 1,0mm Volume 2,0l Weight 0,41kg encumbrance 14 (stats taken from hitchhiker ingame encumbrance is 7 if its not poor fit think)
> 
> How can the bodysuit which has 10 times the thickness of material less weight than the zentai has less warmth than the zentai Also 7 encumbrance for a skintight suit seem a lot considering that lycra should not hinder movement that much basically zentai is kinda too good but lycra body suit iss kinda bad for no reason protection seems a little high though ![image](https://user-images.githubusercontent.com/89189872/244191054-050ae1fb-7b42-47f2-844b-5581c653baa4.png)

Yeah, the zentai is kind of a meme right now with how good it is.

My opinion of that is zentai should have its warmth cut down to 5-10 (if you can see through it without high eye encumbrance, it is really thin and should not provide much warmth), and cut down lycra bodysuit encumbrance to 1 or even 0, but cut the thickness to 0.1-0.5 mm while giving it leggings' and thermal shirt's warmth level of 20.

**Comment by fairyarmadillo** [comment]
> The CDDA item is specifically a biker jacket with scrap metal TIED to it without any holes made in metal. That is even worse than roadsign vest, I'd say.

Then perhaps what we need is to make one or two new improvised armor items to fill gaps in a way that makes both IC and OOC sense.

Any punk rocker is familiar with customizing a leather jacket, and the process does not involve tying things to it with string. Riveting small metal or even plastic plates to a biker jacket could be a way to get a piece of middleweight armor with less time, skill, and resources than it would take to forge and assemble an actual suit of brigandine, and to me it feels like both a sensible thing to do and a way to fill a gap in the armor types which are currently available.

Currently we have a whole slew of pieces that I'm going to refer to as scrap gear - the tire armor, scrap suit, and armored leather jacket, which are almost totally useless because the gambeson, chitin armor, and leather cuirass/armguards exist - I'll refer to these items as mid-tier gear.

Granted the chitin takes a while to craft, but the gambeson is only one rank higher than the scrap armor, it's adjustable to be worn on the normal or outer layer, and it has phenomenal coverage and protection. Its only drawback is that it's too warm, but chitin armor exists as a fantastic alternative.

Most players will take a look at this and decide that it's not worth making any of the scrap gear because by the time they finish working on it, they'll meet the requirements for the mid-tier gear unless they're playing very low int characters. It's a waste of time, calories, and resources.

Given that the scrap gear is basically trash that's beaten and tied together, its skill and proficiency requirements could be reduced, and an intermediate tier of armor could be introduced which bridges the gap. This would make scrap gear more viable for early-game survivors who just need _something_, and they could either try to make the (currently nonexistent) intermediary gear or work toward the higher-quality mid-tier armor.

The mid-tier armor could be like a roadsign vest (a superior version of a scrap cuirass), makeshift brigandine made of a leather jacket with plates riveted on, and maybe something like a leather or denim jacket with patches or padding to approximate a shitty gambeson.

So you'd have a gear crafting progression that was like, thrown-together trash->skilled but still improvised armor->fully custom handmade medieval gear. I'm not saying it needs to be strictly tiered out like that (we're looking for verisimilitude, not video game tiers), but you could have a gradual upward trend in quality as the player gained more skills and tools, rather than players simply skipping all the garbage armor to make the good stuff.

I realize that's out of scope for this issue but you seem like a good person to bring this up with.

**Comment by fairyarmadillo** [comment]
Also @anoobindisguise the red leather jacket and leather jacket probably have different thicknesses because few jackets are actually 1.5mm thick - that's considered a very heavy duty motorcycle jacket and is primarily used by professional riders. Jackets worn strictly for fashion are much thinner. Obviously the encumbrance is way too high on the red jacket, but that's why their thickness varies at least.

**Comment by randomtyper** [comment]
> > The CDDA item is specifically a biker jacket with scrap metal TIED to it without any holes made in metal. That is even worse than roadsign vest, I'd say.
> 
> Then perhaps what we need is to make one or two new improvised armor items to fill gaps in a way that makes both IC and OOC sense.
> 
> Any punk rocker is familiar with customizing a leather jacket, and the process does not involve tying things to it with string. Riveting small metal or even plastic plates to a biker jacket could be a way to get a piece of middleweight armor with less time, skill, and resources than it would take to forge and assemble an actual suit of brigandine, and to me it feels like both a sensible thing to do and a way to fill a gap in the armor types which are currently available.

Hm, then maybe split it into two items. First is where survivor hastily ties a bunch of scrap to the jacket (with all the bad stuff, but VERY quick), and the second is where survivor takes time and skill to use some small sheets, a hammer and a bunch of nails for rivets to create a true postapoc suit with decent protection against zombies. Survivor gear-lite. Not Kevlar, but still decent in its own regard.

> Currently we have a whole slew of pieces that I'm going to refer to as scrap gear - the tire armor, scrap suit, and armored leather jacket, which are almost totally useless because the gambeson, chitin armor, and leather cuirass/armguards exist - I'll refer to these items as mid-tier gear.

Tire armor is a meme. A meme with 20 blunt protection. Scrap suit is in pretty decent spot. Less than a day to make with no proficiencies, a good way to get a passable armor set and those proficiencies. And while it does not have as much upgrade potential as sheet metal armor, initially it has better protection. As for armored jacket, see my first paragraph in the message.

> Granted the chitin takes a while to craft, but the gambeson is only one rank higher than the scrap armor, it's adjustable to be worn on the normal or outer layer, and it has phenomenal coverage and protection. Its only drawback is that it's too warm, but chitin armor exists as a fantastic alternative.

There is a reason why gambesons were so popular in medieval times. Cheap, effective and efficient. And it can't be adjusted as of now, btw, filling both normal and outer layers. The function was obsoleted. Per this: https://github.com/CleverRaven/Cataclysm-DDA/pull/64114

> Most players will take a look at this and decide that it's not worth making any of the scrap gear because by the time they finish working on it, they'll meet the requirements for the mid-tier gear unless they're playing very low int characters. It's a waste of time, calories, and resources.

Metalworking proficiencies are not taught by gambeson making. Just saying. And you would like to get proficiencies before starting to make custom plate, as IIRC making barely understandable recipes with 0 proficiencies is not a good way to get those. And that plate is usually so long to craft you will have plenty of time to run around in scrap, or even case harden your sheet metal stuff. It's not like you don't have alternatives in form of Kevlar gear, though. You can make scrap stuff just to not get killed by that soldier zombie and loot him. You can get a high protection piece that will serve you for quite a while that way at little cost instead of spending 40+ hours stitching that gambeson. And you leveled your metalworking proficiencies on the way, too!

> Given that the scrap gear is basically trash that's beaten and tied together, its skill and proficiency requirements could be reduced, and an intermediate tier of armor could be introduced which bridges the gap. This would make scrap gear more viable for early-game survivors who just need _something_, and they could either try to make the (currently nonexistent) intermediary gear or work toward the higher-quality mid-tier armor.

> The mid-tier armor could be like a roadsign vest (a superior version of a scrap cuirass), makeshift brigandine made of a leather jacket with plates riveted on, and maybe something like a leather or denim jacket with patches or padding to approximate a shitty gambeson.

Roadsigns would make for very crappy armor. Large, thick, made of aluminum or its alloy. Not really protective and "a bit" obstructive. For mid tier plate we have sheet metal armor that can be case hardened. But it has about the same encumbrance as scrap armor and worse protection before case hardening. And sheet metal helmets have 5 mm cotton padding yet only occupy middle layer for some reason... And I would put the scrap armor (despite being called scrap) into the mid tier. It is decent. Not perfect

> So you'd have a gear crafting progression that was like, thrown-together trash->skilled but still improvised armor->fully custom handmade medieval gear. I'm not saying it needs to be strictly tiered out like that (we're looking for verisimilitude, not video game tiers), but you could have a gradual upward trend in quality as the player gained more skills and tools, rather than players simply skipping all the garbage armor to make the good stuff.

The problem is that completely trash armor is often can be worse than no armor. For example, on medieval battlefield (until late medieval era, when thick gambeson became low grade armor and chainmail with brigandines became mid-grade, being pushed out by plate for elites) you could see low grade soldiers in only tunics with shields in hand (or with no armor at all, only weapons at hand) instead of them having armor improvised from wood. It's not that they didn't have sense of self-preservation, it's that they deemed no protection (or only having shield) better than bad protection. There is a line where "improvised" becomes "useless trash". For zombie apocalypse anything not protecting 100% (or as close to that number as possible) of your body is mid at best. It's why Hub's modular armor is often graded quite low by players despite being really high-tech and having decent material and thickness stats along with extra capabilities and MODULARITEH, and plate armor is beloved by most despite having no pockets, needing weeks to make and serving exclusively as protection set with no extra capabilities (warmth? no; enviro? nada; acid? nay; heat? nope).

> I realize that's out of scope for this issue but you seem like a good person to bring this up with.

**Comment by fairyarmadillo** [comment]
There are cases where complete trash might be situationally useful, for instance imagine you have to bike across an early-game city and you don't want to get rocked to death by ferals, but the fact that you're finding army gear everywhere kind of obviates it.

Part of the problem is also how quickly you fly up in skill levels. 4 tailoring is not really a harder requirement than 3. Proficiencies do a good job of filling progression out, but in most of my playthroughs it's gambeson/leather -> chitin -> surivivor/plate and I'm done. It'd be nice if there were more meaningful milestones along the way.

**Comment by randomtyper** [comment]
> There are cases where complete trash might be situationally useful, for instance imagine you have to bike across an early-game city and you don't want to get rocked to death by ferals, but the fact that you're finding army gear everywhere kind of obviates it.

Here, you said it yourself. Trash gear MIGHT be situationally useful. Even if it is viable, it is a stopgap solution at best. And many players prefer to skip such stopgap solutions. Even if one crafts such gear, it is mostly just to get some better gear, so a sally-out or two. After all, unless you know you need at least some armor RIGHT NOW and at any cost, why not wait a bit and get armor that is, you know, useful in most scenarios?

> Part of the problem is also how quickly you fly up in skill levels. 4 tailoring is not really a harder requirement than 3. Proficiencies do a good job of filling progression out, but in most of my playthroughs it's gambeson/leather -> chitin -> surivivor/plate and I'm done. It'd be nice if there were more meaningful milestones along the way, or crafted alternatives that were worth pursuing.

For alternatives worth pursuing... Realistically, for zed rush there is nothing like good plate armor with good underarmor. For specific situations like gun-armed bandits and turrets army gear with ballistic inserts is the top dog. *pulls out 6 mm plate chestplate* And even then, 46 ballistic protection is plenty, especially if you have a kevlar underarmor piece. After all, not everyone is going to pull a .50 cal on you (and .50 packs enough punch to deal with army gear anyways...). 

Sure, you could get that up-armored jacket. But why, really? 

Less encumbrance? Well-made plate armor is pretty unencumbering both IRL and in-game, especially light plate. And protection difference is pretty drastic if the plates on jacket are not overlapped, inviting all sorts of Bri'ish shank-happy lads. Just get a brigandine at this point. It's not DnD where you can wear all-round inferior armor for mobility boost (like (chain) mail vs plate, despite IRL fitted plate amor is superior to mail even in terms of mobility, as you don't have dangly bits, and plate armor does not have unnecessary overlaps and gaps, making protection uniform and allowing to make armor piece lighter). Large plate armor will always be superior to segmented one. 

Pockets? Well, I can see your argument for dusters and trenchcoats, but jacket? Get a load-bearing vest, leg pouches, arm pouches, backpack you can drop (or a dive bag if you want something small, unencumbering yet capable of fitting long stuff) and get over that. You shouldn't really combine full-scale looting and fighting anyways. And if you do that, you likely have a backpack you can just drop, and between backpack volume and fact there is enough gear for strapped layer to carry most of stuff you may need RIGHT NOW, extra 2-4 liters from armored jacket won't really matter.

I understand what you want to see, but at the same time, from realistic standpoint... Why. Plated jackets, trenchcoats and dusters are cool, but will still give more encumbrance than proper plate armor of same thickness and protective value (and better coverage at the same time) even when empty. They will never be a pinnacle of protection, yet will be mediocre in other aspects. Hell, even brigandine is a valid alternative. You could theoretically add some pockets to it, but adding extra pair of liters to your capacity is barely worth it over getting a proper tac vest.

That said, there may be an option. See those Kevlar bodysuits? Apply the same philosophy to the outerwear (dusters, trenchcoats, leather jackets) and it may give some decent results. You won't get much out of steel-only based protection if you are not going full plate, but combining it with Kevlar may produce some decent for its weight armor. And it makes sense as top-of-the-line outerwear, requiring advanced materials, but not as much time investment as plate.

**Comment by fairyarmadillo** [comment]
Again I think you're missing my point. I never said a plated jacket should be better or even as good as plate mail. Given that it amounts to improvised brigandine, it should therefore be worse than brigandine in pretty much every respect except that it's easier to assemble. Something does not have to be as crappy as scrap gear to fill that role, there's an in-between spot it could happily occupy.

**Comment by randomtyper** [comment]
> Again I think you're missing my point. I never said a plated jacket should be better or even as good as plate mail. Given that it amounts to improvised brigandine, it should therefore be worse than brigandine in pretty much every respect except that it's easier to assemble. Something does not have to be as worthless as scrap gear to fill that role, there's an in-between spot it could happily occupy.

A valid criticism in first sentence. When you said "It'd be nice if there were more meaningful milestones along the way, or crafted alternatives that were worth pursuing", by "alternatives worth PURSUING (not just making if you can)" I understood "gear that is not just a stopgap before getting better stuff".  I admit I focused on that part too much. BUT. 

The problem with "milestones along the way" is that they are just milestones. Scrap gear is quick to assemble and costs little to make, it can be made really quickly and having to replace it does not feel like a loss of time, more like "it served its job, now I shall wear proper gear". Purpose made gear is high tier gear, it does not need replacement, you use it when you want and/or can. And, of course, there is scavenged gear of varied quality, something that can either be an upgrade to scrap gear (and something that scrap gear was made to obtain, case in point - US ballistic vest) or a stepping stone to high tier gear (case in point - wetsuits and turnout/fire resistant gear). 

And there is that awkward middle plated jacket will likely get stuck in (and is stuck in currently): too expensive to be easily discarded, yet is not fully qualified to be an end game item and possibly replaceable by scavengeable gear, the exact same problem that plagues current mid-tier craftable gear. Yes, you may have spent 10 hours crafting that jacket. But you have just met a soldier zombie you could beat with scrap gear, and now 8 hours of work (because you could spend just 2 or so hours crafting scrap armor) and some materials just went to nowhere because you got an armor piece with better encumbrance, protection AND carrying capacity. Don't just think in terms of options available to craft, think in terms of options available to player in general.

> Unfortunately this is starting to feel like bikeshedding. If I get some time I'll just PR the stuff myself later and we'll see where it goes.

Will be happy to look into your thought process on practice. But I have to warn you that the concept you have about "mid-tier gear players will want to craft over scavenging or waiting until better gear" may not be as easy to implement. As it stands now, most likely you will just end up making a novelty/optional piece of gear, an item player will most likely only make as personal style choice.

Sorry for being a buzzkiller, I just don't want you making the PR and then being utterly shut down because "idea has no value nor sense, nor is necessary for game". While I don't really agree with default notion "don't add it" (CDDA is a roguelike, it is supposed to have many ways of interacting with world and huge inherent variety that should be expanded when possible and reasonable), having gear that sits in an awkward middle spot where it is both too expensive for short term use yet not useful enough for long term use is not really beneficial for variety.

**Comment by fairyarmadillo** [comment]
> "idea has no value nor sense, nor is necessary for game".

This is kind of rude and I don't think you really get what I'm talking about anyway.

**Comment by Maxab1989** [comment]
I always order leather jackets for my firm from Italy. There are a lot of well-known brands like [Novido](https://novido.it/), Barone, etc that you can trust on for genuine leather products.
